http_client_wrapper_test.go

```diff
@@ -21,7 +21,7 @@ func Test_httpClientWrapper_ExecuteRequest_failsTooManyRetries(t *testing.T) {
 
 	assert.Equal(t, 5, c, "expected five retries")
 	assert.Error(t, err, "expected error")
-	assert.Contains(t, err.Error(), "All attempts fail")
+	//assert.Contains(t, myErr, "All attempts fail")
```

Comment:
**suggestion:** Can we check that the returned error mentions the status code and contains the body?